### DISCUSSION TITLE
clipboard

### DISCUSSION THREAD

**Discussion by twprh** [rustdesk/rustdesk#8929] [General]
Given: Nightly Builds
Question: For some time (imho 1 or 2 weeks ago) the transfer of the clipboard from the controlled to the controller no longer works.
Is this intentional? I found it very useful to be able to copy text on the remote device and then process it locally.

**Comment by fufesou** [comment]
Yes, copying text from controlled side (Windows) to control side is broken in `2024-07-29` nightly build.

Please try the latest nightly build again.

**Reply by twprh** [reply]
thanks for fast answering 
big big thanks for even faster debugging
great job!
